components/ImprovedSearch/RefinementList.tsx

```diff
@@ -53,7 +53,9 @@ export const RefinementList = (props: RefinementListProps) => {
         </EdsProvider>
       ) : (
         <NoRelevant>
-          <FormattedMessage id="asset.no.relevant.filters" defaultMessage="No assets for this filter" />
+          <Typography group="paragraph" variant="meta" color="Tertiary">
```

Comment:
If you inspect using dev tools you will see that `color="tertiary"` doesn't work

Reply:
yeah ... i  kept it to ask ...if this is something we can use through token colors